xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/edit.less

```diff
@@ -79,6 +79,12 @@
   width: 100%;
 }
 
+// XWIKI-16344: Use .xform style standard in the wiki and wysiwyg edit modes
+// Fix for the content's margin-bottom which creates a big vertical gap.
+#contentMeta {
+  margin-bottom: 0px;
```

Comment:
The unit can be omitted when the value is 0 (0px or 0em or 0pt is the same 0).

Reply:
Ok got it.